test/TestConfig/resources/excludes/latest_exclude_SE80.txt

```diff
@@ -23,4 +23,8 @@
 # Exclude tests temporarily
 
 org.openj9.test.vmArguments.VmArgumentTests:testCrNocr																	244 generic-all
-j9vm.test.softmx.SoftmxRemoteTest																						480 generic-all
+j9vm.test.softmx.SoftmxRemoteTest
+
+# Exclude this while investigating https://github.com/eclipse/openj9/issues/2047
+# Intermittent NullPointerException in TestAttachErrorHandling.deleteIpcDirectories
+org.openj9.test.attachAPI.TestAttachErrorHandling:test_startupShutdownRobustness																						480 generic-all
```

Comment:
it should be `2047` not `480`. We can remove line `Exclude this while investigating https://github.com/eclipse/openj9/issues/2047`